ts/src/bitget.ts

```diff
@@ -3490,7 +3492,7 @@ export default class bitget extends Exchange {
             'symbol': market['id'],
             'marginCoin': market['settleId'],
         };
-        const response = await this.privateMixGetPositionSinglePosition (this.extend (request, params));
+        const response = await this.privateMixGetPositionSinglePositionV2 (this.extend (request, params));
```

Comment:
@sc0Vu don't we need to update anything inside `parsePosition` ?

Reply:
@carlosmiei probably not. it looks like they added liquidationPrice in v2 api. Here are diff for their documentation. We already have liquidationPrice in `parsePostion`.

diff for singlePosition api response (left v1, right v2)
<img width="1210" alt="image" src="https://user-images.githubusercontent.com/10494397/236107096-6015c69e-1e6a-4431-a16d-da103bfb3ec5.png">

diff for allPosition api response (left v1, right v2)
<img width="1188" alt="image" src="https://user-images.githubusercontent.com/10494397/236107275-a61dd4bf-8bb3-4d9b-851d-a5e15aa198f5.png">